README.md

```diff
@@ -651,24 +651,37 @@ data products, find the true domain owner of a field attribute)
 
  Create a data contract from the given source location. Prints to stdout.
 
-╭─ Options ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
-│ *  --format        [sql|avro|glue|bigquery]  The format of the source file.                                                     │
-│                    [default: None] [required]                                                                                   │
-│ *  --source        TEXT             The path to the file or Glue Database that should be imported. [default: None] [required]   │
-│    --help                           Show this message and exit.                                                                 │
-╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ *  --format             [sql|avro|glue|bigquery]  The format of the source file. [default: None] [required]                                                                            │
+│ *  --table              TEXT                      List of table ids to import from the bigquery API (repeat for multiple table ids). [default: None] [required]                        │
+│ *  --bt-project-id        TEXT                      The id of the project that we should query on bigtable. [default: None] [required]                                                 │
```

Comment:
I'd use `--bigquery-project`, `--bigquery-dataset`, and `--bigquery-table` as the parameters. And in the description, we should not use bigtable, but bigquery instead. We should make `--bigquery-table` optional and import all tables within the `--bigquery-dataset` when it is missing.

Reply:
Because of how the CLI help works, having a prefix similar to the format helps to group the parameters together that are specific to the import format. In the future, we will probably move towards separate importers, with their very own set of parameters. With this, we can get rid of these prefixes at some point. But this is not too relevant right now.

Reply:
Just to follow up on it: I deliberately named `table` without a prefix, as I imagined it might come in handy for others importers as well, assuming that other formats also allow to have multiple models. But I guess we can change things when that comes around